src/state_control/state_control_Lara.cpp

```diff
@@ -3326,27 +3332,35 @@ int StateControl_LaraDoTwoHandWeaponFrame(struct entity_s *ent, struct ss_animat
         {
             case ITEM_SHOTGUN:
                 weapon = getShotgun();
+                ent->character->currentAmmunitionInventoryItemId = ITEM_SHOTGUN_AMMO; // TODO multiple ammo in TR4

```

Comment:
ITEM_SHOTGUN_AMMO1, 2, 3, 4... and it will be fixed ))

Reply:
ITEM_SHOTGUN_NORMAL_AMMO, ITEM_SHOTGUN_WIDESHOT_AMMO and others are existing yet, but I think that there is no way to select them for now.